posta/queue.py

```diff
@@ -0,0 +1,26 @@
+import traceback
+
+from celery import shared_task, group
+from celery.utils.log import get_task_logger
+
+logger = get_task_logger('posta:coda')
+
+
+@shared_task(ignore_result=True)
+def process_queue(limit=None):
```

Comment:
docstring

Reply:
Non un problema, piu' per le prossime volte: possiamo mantenere i nomi di funzione in italiano? Non e' per niente PEP-8, chiaramente, solo che questo e' possibilmente uno dei casi dove la consistenza batte la correttezza

Reply:
scusami non mi viene proprio, mi impegnerò :)